lib/dir_model/import.rb

```diff
@@ -90,11 +90,9 @@ def find_match
     end
 
     def get_regexp
-      if foreign_value
-        Regexp.new(self.class.options[:regex].call(foreign_value), Regexp::IGNORECASE)
-      else
-        self.class.options[:regex].call
-      end
+      args = []
+      args << foreign_value if foreign_value
+      instance_exec(*args, &self.class.options[:regex])
```

Comment:
If you remove `Regexp::IGNORECASE` you need to revised all the regex on finalcloud, problem come from string interpolation